ui/packages/shared/components/src/KeyDownContext/index.tsx

```diff
@@ -39,7 +54,17 @@ const useIsShiftDown = (): boolean => {
     };
   }, []);
 
-  return isShiftDown;
+  const value = useMemo(() => ({isShiftDown}), [isShiftDown]);
+
+  return <KeyDownContext.Provider value={value}>{children}</KeyDownContext.Provider>;
+};
+
+export const useKeyDown = (): KeyDownState => {
+  const context = useContext(KeyDownContext);
+  if (context == null) {
+    return DEFAULT_VALUE;
+  }
+  return context;
 };
 
-export default useIsShiftDown;
+export default KeyDownContext;
```

Comment:
Looks like you forgot to use this context provider. Maybe you can add it to the App.tsx file in the `packages/app/web/src` folder

Reply:
Ah, sorry missed the `MetricsGraph`, now wrapped the whole `ProfileExplorer`.  

Trying to minimize the side effect changes on the cloud.